go.mod

```diff
@@ -3,15 +3,13 @@ module github.com/ConsenSys/quorum-go-utils
 go 1.13
 
 require (
-	github.com/ConsenSys/quorum/crypto/secp256k1 v0.0.0-20201109194625-1ecd42625e8e
+	github.com/consensys/goquorum-crypto-secp256k1 v0.0.1
+	github.com/ethereum/go-ethereum v1.10.8 // indirect
```

Comment:
this is an issue - we need to make sure that neither go-ethereum nor goquorum are in the dependency list otherwise we have to use lgpl license. can you check if github.com/consensys/goquorum-crypto-secp256k1 is puling that dependency in.